tests/test_matrix_cross_signing.py

```
@@ -0,0 +1,562 @@
+
+    async def json(self, *, content_type: object = None) -> dict[str, object]:
+        del content_type
+        master_keys: dict[str, object] = (
+            {self.user_id: {"keys": {}}} if self.has_master else {}
+        )
+        return {"master_keys": master_keys}
+
+
+class _GuardedCrossSigningClient(_CrossSigningClient):
+    user_id = "@bot:example.org"
+    access_token = "token"
```

Ruff (0.16.0)

[error] Possible hardcoded password assigned to: "access_token"

(S105)